config.sample.yml

```diff
@@ -0,0 +1,16 @@
+# RabbitMQ connection URL
+registryUrl: 'amqp://guest:guest@rabbitmq:5672/'
+
+# RabbitMQ exchange name (cron-tasks by default)
+# exchangeName: 'cron-tasks'
+
+# List of tasks to manage
+tasks:
+  # Worker type to add tasks (equal to the Registry queue name)
+  - workerType: 'archiver'
```

Comment:
isn't it cron-tasks/archiver?
Maybe we would like to have another archiver queue in Future, so it would cause name collision